dapt-reqs.html

```diff
@@ -0,0 +1,636 @@
+                    <th colspan="1" rowspan="1">Applies to Dubbing?</th>
+                    <th colspan="1" rowspan="1">Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td colspan="1" rowspan="1">Times</td>
+                    <td colspan="1" rowspan="1"><dfn>PS1</dfn>. Identify key times in programme dialog, mark gaps
+                    <td colspan="1" rowspan="1">X
+                    <td colspan="1" rowspan="1">X
+                    <td colspan="1" rowspan="1">
+                        <ul>
+                            <li>Automatically or manually process the programme audio track
+                                to identify intervals either with spoken dialogs
+                                to be translated or within which description audio may be inserted.</li>
+                        </ul>
+                </tr>
+                <tr>
+                    <td colspan="1" rowspan="4">Text
+                    <td colspan="1" rowspan="1"><dfn>PS2a</dfn>. Describe images (AD)
+                    <td colspan="1" rowspan="1">X
+                    <td colspan="1" rowspan="1">
+                    <td colspan="1" rowspan="1"><ul>Write a set of descriptions to fit within the identified gaps.</li></ul>
+                </tr>
+                <tr>
+                    <td colspan="1" rowspan="1"><dfn>PS2b</dfn>. Transcribe original text (DUB)
+                    <td colspan="1" rowspan="1">
+                    <td colspan="1" rowspan="1">X
+                    <td colspan="1" rowspan="1">
+                        <ul>
+                            <li>Transcribe the dialogue in the original language to create a source transcription text</li>
+                            <li>Notate dialogue events with character information and other annotations</li>
+                            <li>Generate localization notes to guide further adaptation</li>
+                        </ul>
+                    </td>
+                </tr>
+                <tr>
+                    <td colspan="1" rowspan="1"><dfn>PS2c</dfn>. Translate text (DUB)
+                    <td colspan="1" rowspan="1">
+                    <td colspan="1" rowspan="1">X
+                    <td colspan="1" rowspan="1">
+                        <ul>
+                            <li>Translate the dialogue to a target language</li>
+                        </ul>
+                    </td>
+                </tr>
+                <tr>
+                    <td colspan="1" rowspan="1"><dfn>PS2d</dfn>. Adapt text (DUB)
+                    <td colspan="1" rowspan="1">
+                    <td colspan="1" rowspan="1">X
+                    <td colspan="1" rowspan="1">
+                        <ul>
+                            <li>Adapt the translation to the dubbing; ex.
+                                <span class="ednote"> What does ex. mean?</span>
```

Comment:
Sorry it's a French abbreviation. It means "for example". We can replace it with "e.g."?

Reply:
Done in d8b74a9.